src/libraries/Native/build-native.proj

```diff
@@ -15,7 +16,7 @@
 
   <Target Name="BuildNativeUnix"
           BeforeTargets="Build"
-          Condition="'$(TargetOS)' != 'Windows_NT'">
+          Condition="!$([MSBuild]::IsOsPlatform(Windows))">
```

Comment:
Unsure if this ever worked but the TargetOS check doesn't assert the current OS but the target one that is passed in via the `-os` switch.

Reply:
Can I build the native bits targeting Unix on Windows?

Reply:
This broke with the WASM on Windows build, since it was going down the `build-native.sh` path when it should have gone down the `build-native.cmd` path.